config/default.cfg

```diff
@@ -30,7 +30,7 @@ cfg$extramappings_historic <- ""
 cfg$inputRevision <- "6.326"
 
 #### Current CES parameter and GDX revision (commit hash) ####
-cfg$CESandGDXversion <- "2179845fc48aa767d13da5538296f11e5a8a756e"
+cfg$CESandGDXversion <- "e9936f817c50b06b75f5ba21a14204ff08590ced"
```

Comment:
`2179845` is from revision 6.326, `mrremind` version 0.151.2, from [MR #335](https://github.com/pik-piam/mrremind/pull/335).
`e9936f8` is from revision 6.324IEA-buildings-NavUpdate, `mrremind` version 0.147.1.9001, which is some unkown state after [MR #328](https://github.com/pik-piam/mrremind/pull/328).

We have no way an systematically analysing the differences between input data revisions, but [MR #335](https://github.com/pik-piam/mrremind/pull/335) "updated cement production trajectories", which differ quite substantially between the two revisions (90 % in `CHA` in 2035).

So while the NDC scenario will not be infeasible with this .gdx, the results will be wrong and meaningless.

Reply:
And therefore input .gdx input data and CES parameters are inconsistent.